src/nvim/globals.h

```diff
@@ -990,8 +988,8 @@ EXTERN char e_api_error[] INIT(=N_("E5555: API call: %s"));
 
 EXTERN char e_luv_api_disabled[] INIT(=N_("E5560: %s must not be called in a lua loop callback"));
 
-EXTERN char e_floatonly[] INIT(=N_(
-                                  "E5601: Cannot close window, only floating window would remain"));
+EXTERN char e_floatonly[] INIT(=
+                                 N_("E5601: Cannot close window, only floating window would remain"));
```

Comment:
this should just be moved to the previous line. is this configurable or more like a uncrustify bug/limitation?

Reply:
It's more that it tries (and fails) to properly split the line since it's over the 100-character limit. We have following options, from what I believe to be from the best to worst:

1. Add a `\` after `INIT` for all lines exceeding 100 characters. This would typically look like
```c
EXTERN char e_floatonly[] \
  INIT(=N_("E5601: Cannot close window, only floating window would remain"));
```

2. Revert the offending commit, meaning resetting `sp_enum_after_assign = ignore`. I've rechecked and the number of changes is really negligible so this isn't a big deal.

3. Disable the formatting for the `INIT`-graveyard with a `// uncrustify:off` before and `// uncrustify:on` after.

4. Increase the character-limit to 130

Pick one.